src/Uno.Extensions.Core.Generators/PropertySelector/Rules.PropertySelector.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Uno.Extensions.Generators.Common;
+
+namespace Uno.Extensions.Reactive.Generator;
+
+internal static partial class Rules
+{
+	public static class PS0001
+	{
+		private const string message = "'{0}' in '{1}' is not a property member."
+			+ " Property selectors can only be of the form `e => e.A.B.C`, you cannot use method nor external value (i.e. cannot have any closure), and cannot be a method group.";
+
+		public static readonly DiagnosticDescriptor Descriptor = new DiagnosticDescriptor(
+			nameof(PS0001),
+			"A property selector can only use property members",
+			message,
+			Category.Usage,
+			DiagnosticSeverity.Error,
+			helpLinkUri: "https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/PropertySelector/rules.html#PS0001",
```

Comment:
Let's use an AKA instead.
```suggestion
			helpLinkUri: "https://aka.platform.uno/PS0001",
```